src/rest.cpp

```diff
@@ -181,7 +181,7 @@ static bool rest_headers(const std::any& context,
     headers.reserve(count);
```

Comment:
6fb65b49f4: the scripted diff will modify files that are not tracked by git. Could use `...  -- $(git grep -l Ensure)`